war/src/main/js/add-item.js

```diff
@@ -0,0 +1,292 @@
+// Initialize all modules by requiring them. Also makes sure they get bundled (see gulpfile.js).
+var $jq = require('jquery-detached').getJQuery();
+
+var getItems = function(root){
+  var $ = $jq;
```

Comment:
? why not just $ globally? but for tests, it seemed to work best for me to use:
var jQuery = require('jquery-detached'); // global
and
var $ = jQuery.getJQuery(); // per function

though I would much prefer a $ global to the file...